src/main/java/uk/gov/di/ipv/lambda/CredentialIssuerHandler.java

```diff
@@ -27,8 +28,16 @@
 
     private final CredentialIssuerService credentialIssuerService;
 
-    protected static final CredentialIssuerConfig PASSPORT_ISSUER = new CredentialIssuerConfig("PassportIssuer", URI.create("http://www.example.com"));
-    protected static final CredentialIssuerConfig FRAUD_ISSUER = new CredentialIssuerConfig("FraudIssuer", URI.create("http://www.example.com"));
+    protected static final CredentialIssuerConfig PASSPORT_ISSUER = new CredentialIssuerConfig(
+            "PassportIssuer",
+            URI.create("http://www.example.com"),
```

Comment:
Should we be loading these from environment variables? We could put some values in the configurationService class for this to help with either connecting to PaaS or to localstack depending on how it's ran?

Reply:
I think Phil has written a story to think about how to deal with this kind of configuration. For now we're just hardcoding to get the flow up and running, and can make it configurable later on.

Reply:
Ok cool sounds good 👍